src/error-boundary/FullScreenErrorView.tsx

```diff
@@ -59,6 +57,7 @@ export function FullScreenErrorView({onRestartApp, errorCode}: ErrorProps) {
 const useStyles = StyleSheet.createThemeHook((theme) => ({
   safearea: {
     flex: 1,
+    paddingTop: 44,
```

Comment:
it is possible to remove this? seems like a hack, maybe wrap it in <safearea> component?

Reply:
We could but then the screen will be moved up by 44 units up, I guess it's okay since we don't expect users to see this screen too often.

Reply:
I wrote this link https://reactnative.dev/docs/safeareaview in my previous message but seems like it was not published.

Reply:
There is one already, but perhaps not for that view.

Reply:
What if I move the padding to the `container` and not `safeArea`?

Reply:
<img src="https://github.com/AtB-AS/mittatb-app/assets/1777333/41a8b38c-0941-424e-9c2c-ecfdc9071e40" width="350" />

<img src="https://github.com/AtB-AS/mittatb-app/assets/1777333/3a1ef8e0-fb60-4519-89e3-002a4bda274c" width="350" />

Comparison without extra padding